gcsmigrate/README.md

```diff
@@ -1,18 +1,18 @@
-# Migrating from mogile to S3
+# Migrating from mogile to GCS
 
-This folder contains code for migrating from Mogile to S3.
+This folder contains code for migrating from Mogile to GCS.
 
-In broad strokes, the migration from mogile to S3 consists of the following steps:
+In broad strokes, the migration from mogile to GCS consists of the following steps:
 
 1. Build and configure the migration.
-1. Sync data from mogile to an AWS bucket
+1. Sync data from mogile to an GCS bucket
 1. Remove or fix bad data.
-1. Run a large-scale migration in AWS lambda.
+1. Run a large-scale migration in a cloud function.
 1. Stop ingesting new data into contentrepo.
-1. Run a large-scale migration in AWS lambda.
+1. Run a large-scale migration in a cloud function.
```

Comment:
Are there two "large-scale" migrations? I thought the second one was supposed to be a much smaller incremental sync.

Reply:
I'm going to delete this content and move it to a confluence page.